accounts/templates/registration/sign_up.html

```diff
@@ -3,28 +3,43 @@
 {% load static %}
 
 <main id="main">
-  <div class="container">
-    <h2>Sign Up</h2>
-    {{ form.non_field_errors }}
-    <form method="post">
-      <!-- prettier-ignore -->
-      {% csrf_token %}
-      {% for field in form %}
-      {{ field.errors }}
-      <div class="form-group row">
-        <label class="col-sm-3 col-form-label" for="{{ field.id_for_label }}">
-          {{ field.label }}
-        </label>
-        <div class="col-sm-5">
-          {{ field }}
-          <small class="form-text text-muted">
-            {{field.help_text}}
-          </small>
+  <div class="container" id = "sign-up">
+    <div class = "row">
+      <div class="col-md-6 col-sm-12 col-xs-12">
+        <h1 >Sign Up</h1>
+        <div class = "form-sign">
+          {{ form.non_field_errors }}
+          <form method="post">
+            <!-- prettier-ignore -->
+            {% csrf_token %}
+            {% for field in form %}
+            {{ field.errors }}
+            <div class="form-group">
+              <label class="col-form-label" for="{{ field.id_for_label }}">
+                {{ field.label }}
+              </label>
+              <div class="">
+                {{ field }}
+                <small class="form-text text-muted">
+                  {{field.help_text}}
+                </small>
+              </div>
+            </div>
+            {% endfor %}
+            <label class="check-terms"><input type="checkbox" value="" style="width: fit-content;"> I Agree to the <a>Terms &amp; Conditions</a>.</label>
```

Comment:
Could you make this checkbox mandatory to click before a user can click on "Create Account"?

Reply:
Must've left it, updated.